docs/src/private/basis.md

```diff
@@ -4,6 +4,10 @@
 LFAToolkit.AbstractBasis
 LFAToolkit.gaussquadrature
 LFAToolkit.gausslobattoquadrature
+LFAToolkit.transformquadrature
+LFAToolkit.sausage
+LFAToolkit.kosloff_tal_ezer
+LFAToolkit.hale_trefethen_strip
```

Comment:
Two notes here

1) Julia (tragically) prefers `koslofftalezer` over `kosloff_tal_ezer`, but I think this is a reasonable place to prefer underscores
https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/

2) Do we want to add the word transformation to make the purpose of these functions more clear? i.e. `sausage_transformation' and `kosloff_tal_ezer_transformation`?